vars/functionalPackages.groovy

```diff
@@ -0,0 +1,41 @@
+// vars/functionalPackages.groovy
+
+/**
+ * functionalPackages.groovy
+ *
+ * functionalPackages variable
+ */
+
+/**
+ * Method to return the list of packages to install for functional testing
+ */
+
+String call(Integer client_ver, String next_version) {
+    return functionalPackages(hwDistroTarget(), client_ver, next_version)
+}
+
+String call(String distro, Integer client_ver, String next_version) {
+    String daos_pkgs = getDAOSPackages(distro, next_version)
+    String pkgs = " openmpi3 hwloc ndctl fio " +
+                  "patchutils ior-hpc-daos-${client_ver} " +
+                  "romio-tests-daos-${client_ver} " +
+                  "testmpio " +
+                  "mpi4py-tests " +
+                  "hdf5-mpich-tests " +
+                  "hdf5-openmpi3-tests " +
+                  "hdf5-vol-daos-mpich-tests " +
+                  "hdf5-vol-daos-openmpi3-tests " +
+                  "MACSio-mpich " +
+                  "MACSio-openmpi3 " +
+                  "mpifileutils-mpich-daos-${client_ver} "
```

Comment:
Wouldn't we want this list to be stored in the DAOS repository?

Or use a tool to query the spec file?

Reply:
> Wouldn't we want this list to be stored in the DAOS repository?

Perhaps.  Let me see how it can be done usefully.
 
> Or use a tool to query the spec file?

Well, there is nothing to query,  Ultimately, this whole list should be dependencies of the _daos-tests_ RPM so that they are automatically installed when _daos-tests_ is installed.  There is a back-burnered PR open to do this.

Reply:
Fixed in next push.